internal/commands/server/create.go

```diff
@@ -290,6 +290,10 @@ func (s *createCommand) ExecuteWithoutArguments(exec commands.Executor) (output.
 		return nil, fmt.Errorf("a password-delivery method, ssh-keys or a custom image must be specified")
 	}
 
+	if !s.createPassword.Value() && s.params.PasswordDelivery != "none" {
+		_ = s.createPassword.Set("true")
```

Comment:
I'm not sure if I agree with this. I get the idea - if somebody sets the password delivery method then they for sure want password, but it still seems a bit too implicit to me. Not a blocker though. You might want to update the help text for `--password-delivery` flag to make sure this is documented somewhere ;)

Reply:
As it was true by default before, this addition is for not messing with defaults for a minor version upgrade. For future versions we might think if --create-password is even needed. Maybe a --login-type would be better that would default to SSH?

Would you think this would make sense, and we should add a deprecation notice to that?

Reply:
Switching to something like `--login-type` is imo a good idea, as it removes any ambiguity. With that it's clear that you have only one option, and we also get to clearly set the safer default 👍 
In this case, having this ☝️  behavior as a temp solution to avoid breaking change is totally ok to me :)